object-has-acessible-name-8f3b6.md

```diff
@@ -0,0 +1,198 @@
+---
+id: 8fc3b6
+name: Object element in the accessibility tree has non-empty accessible name
+rule_type: atomic
+description: |
+  This rule checks that each object has an accessible name.
+accessibility_requirements:
+  wcag20:1.1.1: # Non-text Content (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility Tree
+acknowledgments:
```

Comment:
Maybe add the source of the Moon speech used in the examples.

Reply:
This file has already been used in other rules so I don't know who to acknowledge.

Reply:
Something like "audio is excerpt from JFK's 'Moon Speech'".